tailwind.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  important: true,
```

Comment:
💚 What is overriding the tailwind styles? Is it worth the time/effort to refactor them out of app to avoid using the important?

Reply:
Agree. not massively against adding this in if its saving headaches, I was just trying to avoid for as long as possible.

I had it turned on originally when I was testing out tailwind and I didn't see any issues from it from a cursory look though!

Reply:
Yeah I will give this a crack 👍

Reply:
Refactored and removed!